service/collector/subnet.go

```diff
@@ -256,8 +298,25 @@ func (e *Subnet) getSubnetInfoFromAPI(ctx context.Context, awsClients clientaws.
 		if subnet.Tags[key.TagInstallation] != e.installationName {
 			continue
 		}
+		subnet.AvailableIPPercentage, err = getAvailableIPPercentage(subnet.Tags["CidrBlock"], subnet.AvailableIPs)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
 		subnets = append(subnets, subnet)
 	}
 	res.Subnets = subnets
 	return &res, nil
 }
+
+func getAvailableIPPercentage(cidr string, availableIps int64) (float64, error) {
```

Comment:
Here we first calculate the total IPs from the CIDR block and from there get the percentage of IPs that are still available.